api/validators/FeedbackControllerRequests.ts

```diff
@@ -1,27 +1,44 @@
 import { Type } from 'class-transformer';
-import { IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
+import { Allow, IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
 import { IsValidFeedbackType, IsValidFeedbackStatus } from '../decorators/Validators';
 import {
   SubmitEventFeedbackRequest as ISubmitEventFeedbackRequest,
   SubmitFeedbackRequest as ISubmitFeedbackRequest,
   UpdateFeedbackStatusRequest as IUpdateFeedbackStatusRequest,
+  FeedbackSearchOptions as IFeedbackSearchOptions,
   Feedback as IFeedback,
   FeedbackType,
   FeedbackStatus,
 } from '../../types';
+import { UserModel } from '../../models/UserModel';
+import { EventModel } from '../../models/EventModel';
 
 export class Feedback implements IFeedback {
   @IsDefined()
   @IsNotEmpty()
-  title: string;
+  user: UserModel;
```

Comment:
We don't really include users as part of API requests; we have access to the user throughout the code itself (if you take a look at the service-level method, there's a user parameter being passed in). This comes from the controller-level method, which makes use of the `@AuthenticatedUser()` decorator. With the structure you have right now, we'd need to incorrectly pass in the entire User object in the API request